src/altinn-app-frontend/src/features/form/containers/RepeatingGroupsEditContainer.tsx

```diff
@@ -123,19 +123,18 @@ export function RepeatingGroupsEditContainer({
     }
   };
 
-  const getNextIndex = (): number | null => {
-    if (!filteredIndexes) {
-      return editIndex < repeatingGroupIndex ? editIndex + 1 : null;
-    } else {
-      const filteredIndex = filteredIndexes.indexOf(editIndex);
-      return filteredIndexes.slice(filteredIndex).length > 1
+  let nextIndex: number | null = null;
+  if (!filteredIndexes) {
```

Comment:
Minor nitpick I noticed now.. The way I read this if statement is 'if not filtered indexes, do this, otherwise do this'. To avoid a double-negative (which can trip you up when reading code), I prefer to swap these around. In some cases (and if this was a function), these `if (!condition)` branches near the top are often used to check for prerequisites, and in those cases I think the [return early pattern](https://medium.com/swlh/return-early-pattern-3d18a41bba8) is a better approach.